plugin/pathvirt/README

```diff
@@ -0,0 +1,95 @@
+# pathvirt (Path Virtualization DMTCP Plugin)
+
+pathvirt is a DMTCP plugin that virtualizes access to file system paths for
+programs run under DMTCP. This allows path access to be transparently
+redirected elsewhere on the file system on DMTCP restart.
+
+## Usage
+
+pathvirt operates on the basis of path prefixes and is interfaced with using
+the DMTCP_PATH_PREFIX environment variable.
+
+On DMTCP init, set DMTCP_ORIGINAL_PATH_PREFIX to a colon delimited list of path
+prefixes to register them with pathvirt. This is required, but does not
+actually change anything about the execution of the program until the first
+restart. For example:
+
+    $ DMTCP_ORIGINAL_PATH_PREFIX=/home/user1/bin1:/home/user1/lib1:/home/user1/doc1 \
+        dmtcp_launch -i4 --with-plugin $DMTCP_ROOT/lib/dmtcp/libdmtcp_pathvirt.so \
+        ./my-program
+
+Then, on any DMTCP restart, you can optionally set DMTCP_NEW_PATH_PREFIX to a
+second colon delimited list of path prefixes. For example:
+
+    $ DMTCP_PATH_PREFIX=/home/user2/bin2:/home/user2/lib2:/home/user2/doc2 \
+        ./dmtcp_restart_script.sh
+
+If you *do not* provide DMTCP_NEW_PATH_PREFIX on restart, pathvirt does nothing,
```

Comment:
`one *does not*`, or `If DMTCP_NEW_PATH_PREFIX is not provided on restart,`

`pathvirt does nothing` -> `pathvirt has no effect`